internal/runner/runner_test.go

```diff
@@ -45,17 +46,32 @@ func TestRun(t *testing.T) {
 	}
 }
 
+func TestRunNoUpdatesCase(t *testing.T) {
+	inBytes, err := ioutil.ReadFile("testdata/no_direct_updates.json")
+	if err != nil {
+		t.Errorf("Failed to read input file: %w", err)
+	}
+	in := bytes.NewBuffer(inBytes)
+	var result bytes.Buffer
+	err = runner.Run(in, &result, true, true, false, runner.StyleDefault)
```

Comment:
```suggestion
	err = runner.Run(in, &result, true, false, false, runner.StyleDefault)
```

no_direct_updates does not have indirect dependencies with updates, so this is not needed